### PR TITLE
Suspend/unsuspend users widgets added to admin panel

### PR DESCRIPTION
**Pull request by Dzeranov**

### CODE REVIEW

File: src/pages/Workspace/admin-panel/suspend-users.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          .catch((error) => notifier.error(error.message))
+          .finally(() => dispatch(finishGlobalLoading()));
+      }}
+    >
+      {(formikObj) => {
+        return (
+          <Form className="suspendUsers">
+            <h4>Suspend Users</h4>
+            <FormControl margin="normal" className="execution__input">
+              <TextField
```
Comment (vkomodey) [inline src/pages/Workspace/admin-panel/suspend-users.jsx:50]: let's add a hint, on how to use this text field. I mean, right now it is impossible to understand that this input accepts multiple emails at once

the same for unsuspend action

File: src/pages/Workspace/admin-panel/suspend-users.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-unused-vars */
```
Comment (vkomodey) [inline src/pages/Workspace/admin-panel/suspend-users.jsx:1]: delete the line and don't use 'unused vars'

**Review by vkomodey — CHANGES_REQUESTED** [review]

File: src/pages/Workspace/admin-panel/suspend-users.jsx (hunk 1 above)
Comment (Dzeranov) [reply src/pages/Workspace/admin-panel/suspend-users.jsx:50]: Sure.

File: src/pages/Workspace/admin-panel/suspend-users.jsx (hunk 2 above)
Comment (Dzeranov) [reply src/pages/Workspace/admin-panel/suspend-users.jsx:1]: Not using, this line was added unintetionally.

**Review by vkomodey — APPROVED** [review]

**Pull request merged**